src/components/Sidebar/BrowseContainer.js

```diff
@@ -0,0 +1,14 @@
+import { connect } from 'react-redux';
+import Browse from './Browse';
+
+
+export function browseSelector(state) {
+  return state.getIn(['browse', 'currentBrowseLevel']);
```

Comment:
I thought we decided not to use a selector for now? If you do want to use it, let's call it `currentBrowseLevelSelector` and actually use it (in line 11 below).

Reply:
I removed the selector